crates/tabby-common/src/index/mod.rs

```diff
@@ -62,7 +82,7 @@ impl IndexSchema {
         let mut builder = Schema::builder();
 
         let field_corpus = builder.add_text_field("corpus", STRING | FAST);
-        let field_source_id = builder.add_text_field("source_id", STRING | FAST);
+        let field_source_id = builder.add_text_field("source_id_v2", STRING | FAST);
```

Comment:
@xxs-wallace we manually change the field name so the whole index will be rebuild automatically